api-reference/openapi.yaml

```diff
@@ -348,6 +348,49 @@ paths:
           $ref: "#/components/responses/notfound"
         "429":
           $ref: "#/components/responses/throttled"
+  "/v3/governance/streets/{usrn}":
+    get:
+      description: Find a street by USRN
+      parameters:
+        - $ref: "#/components/parameters/usrn"
+      responses:
+        "200":
+          description: ""
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    "$ref": "#/components/schemas/streetModel"
+        "404":
+          $ref: "#/components/responses/notfound"
+        "429":
+          $ref: "#/components/responses/throttled"
+  "/v3/governance/streets":
+    get:
+      description: Finds streets
+      parameters:
+        - $ref: "#/components/parameters/limit"
+        - $ref: "#/components/parameters/offset"
+        - $ref: "#/components/parameters/boundedBoxBottomLeft"
+        - $ref: "#/components/parameters/boundedBoxTopRight"
```

Comment:
This doesn't work right now, so let's remove bounding boxes and add them when we can support them